snuba/web/views.py

```diff
@@ -330,11 +332,18 @@ def storage_delete(
         try:
             schema = RequestSchema.build(HTTPQuerySettings, is_delete=True)
             request_parts = schema.validate(body)
-            payload = delete_from_storage(
-                storage,
-                request_parts.query["query"]["columns"],
-                request_parts.attribution_info,
-            )
+            if get_config("use_bulk_deletes", False):
```

Comment:
The default is `False` but shouldn't we set `use_bulk_delete` to `False` using `set_config()` in this PR and flip it to `True` once we are ready to use bulk delete in s4s? Also, the note on https://github.com/getsentry/snuba/blob/ce292773fabd4f8a49970944d794868230655cd3/snuba/state/__init__.py#L190 says that `get_config()` is supposed to be deprecated. Does it make sense to add a function in `snuba/state` that returns `bool` and use it here?

Reply:
oh didn't see that comment - I'll update

and I can set the default to be `0` and use `get_int_config`